### PR TITLE
Fetch upstream

### PR DESCRIPTION
**Pull request by JanDeDinoMan**
This PR pulls upstream BUT:

- Tor is removed
- Hopefully transparency is still working

please run local to confirm still working

### CODE REVIEW

File: config_example.json [hunk 1]
```diff
@@ -15,4 +16,29 @@
...
+            "password": "password",
+            "start_coords": [0, 0]
+        },
+        "worker2username": {
+            "password": "password",
+            "start_coords": [0, 0]
+        }
+    }
+}
+>>>>>>> 982a3908ce67acddb140daa4abcf39015811fe7c
```
Comment (cody-quinn) [inline config_example.json]: This wasn't merged

**Review by cody-quinn — CHANGES_REQUESTED** [review]
Make sure that the auto updating still works aswell, I couldn't find anything relating to it, it wasnt removed was it?

File: config_example.json (hunk 1 above)
Comment (JanDeDinoMan) [reply config_example.json]: Ah good catch .gitignore got me

**Review by cody-quinn — APPROVED** [review]
Looks good

**Pull request merged**